### ISSUE TITLE
Finetuning VGGs on a small dataset

### ISSUE THREAD

**Issue by paganpasta** [paganpasta/eqxvision#28]
Currently, Imagenet pre-trained VGGs show a large gap in top-1 accuracy. The difference could be due to rigidity of the `classifier` layer. To see if the generated features are comparable, the experiment will be to finetune and compare Vgg-11 say on a small dataset like STL-10.
